packages/openapi-generator/src/generator.ts

```diff
@@ -60,7 +63,18 @@ export async function generate(options: GeneratorOptions): Promise<void> {
       uniqueServiceName
     );
   });
-  await Promise.all(promises);
+
+  try {
+    await finishAll(promises);
+  } catch (err) {
+    if (promises.length > 1) {
+      cli.error(
+        new ErrorWithCause('Some clients could not be generated.', err)
+      );
+    } else {
+      new ErrorWithCause('Could not generate client.', err);
```

Comment:
question: why not using `cli.error` for alignment?

Reply:
I just forgot ;) But I now removed it in general.